From 8fbf378f1f1f92ed861548c59fd39cbaaaaa60ef Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 18 Jul 2017 23:27:30 +0800
Subject: [PATCH 2148/5242] MLK-16017 arm64: dts: freescale: imx8qxp: add
 system controller rtc

commit  4a22ed04bed93d8f107ebf300263943d5777f2af from
https://source.codeaurora.org/external/imx/linux-imx.git

Add i.MX8QXP system controller RTC support.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index d48bc99..6939f7a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -99,6 +99,10 @@
 		compatible = "fsl,imx8qxp-iomuxc";
 	};
 
+	rtc: rtc {
+		compatible = "fsl,imx-sc-rtc";
+	};
+
 	timer {
 		compatible = "arm,armv8-timer";
 		interrupts = <GIC_PPI 13 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>, /* Physical Secure */
-- 
1.7.9.5

